src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static Random random = new Random();
+    private static Color color;

```

Comment:
Class variables are harder to maintain and should be used only when we require to share some state for different methods.
Here it should be just local method variable
```suggestion
```